src/ExpressionEvaluator/Core/Source/ResultProvider/ResultProviderPhone.csproj

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <OutDirSuffix>Phone\</OutDirSuffix>
+    <ProjectGuid>{6A4C8A1D-4C47-4191-9681-136FC3B7209C}</ProjectGuid>
```

Comment:
Actually, the Roslyn.sln changes are below...I could have just looked... :)

It seems like the sln file gets the ProjectGuid directly from the project file, so the overwriting *shouldn't* have any negative consequence.